src/main/kotlin/org/opensearch/observability/validation/IntegrationComponent.kt

```diff
@@ -0,0 +1,15 @@
+package org.opensearch.observability.validation
+
+private const val ROOT = "../../../schema/system/"
+
+enum class IntegrationComponent(val resourcePath: String) {

```

Comment:
these validation should be declared under the system catalog context (package ?)